SportsHub/SportsHub.Application/Authentication/Models/DTOs/ArticleResponseDTO.cs

```diff
@@ -0,0 +1,16 @@
+﻿using SportsHub.Domain.Models;
+
+namespace SportsHub.AppService.Authentication.Models.DTOs
+{
+    public class ArticleResponseDTO
```

Comment:
why is it here? response model should be in API layers

Reply:
Where in API layer?